pom.xml

```diff
@@ -226,6 +204,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
+        <version>${maven-jar-plugin.version}</version>
```

Comment:
same here

Reply:
I am just being consistent with your other code in this POM file that already contains redundant versions. If you think the redundancy should be eliminated, remove it from your existing code in a separate PR and then I will follow suit in this PR.